<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="6" hostname="DESKTOP-BIEA88R" failures="3" tests="11" name="Test" time="0.123" errors="0" timestamp="2022-08-22T17:18:52 IST">
  <testcase classname="org.SDET.PreacticeWIthTestNG.CustomerPriority" name="createCustomerTest" time="0.016"/>
  <testcase classname="org.SDET.PreacticeWIthTestNG.Booking" name="bookingTest" time="0.002"/>
  <testcase classname="org.SDET.PreacticeWIthTestNG.Booking" name="bookingTest" time="0.002"/>
  <testcase classname="org.SDET.PreacticeWIthTestNG.Booking" name="bookingTest" time="0.003"/>
  <testcase classname="org.SDET.PreacticeWIthTestNG.Booking" name="bookingTest" time="0.002"/>
  <testcase classname="org.SDET.PreacticeWIthTestNG.Booking" name="bookingTest" time="0.001"/>
  <testcase classname="org.SDET.PreacticeWIthTestNG.Booking" name="bookingTest" time="0.001"/>
  <testcase classname="org.SDET.PreacticeWIthTestNG.Booking" name="bookingTest" time="0.001"/>
  <testcase classname="org.SDET.PreacticeWIthTestNG.CustomerInvocationCount" name="createCustomerTest" time="0.001">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="Index 5 out of bounds for length 3">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 5 out of bounds for length 3
at org.SDET.PreacticeWIthTestNG.CustomerInvocationCount.createCustomerTest(CustomerInvocationCount.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- createCustomerTest -->
  <testcase classname="org.SDET.PreacticeWIthTestNG.CustomerInvocationCount" name="createCustomerTest" time="0.001">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="Index 5 out of bounds for length 3">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 5 out of bounds for length 3
at org.SDET.PreacticeWIthTestNG.CustomerInvocationCount.createCustomerTest(CustomerInvocationCount.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- createCustomerTest -->
  <testcase classname="org.SDET.PreacticeWIthTestNG.CustomerInvocationCount" name="createCustomerTest" time="0.002">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="Index 5 out of bounds for length 3">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 5 out of bounds for length 3
at org.SDET.PreacticeWIthTestNG.CustomerInvocationCount.createCustomerTest(CustomerInvocationCount.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- createCustomerTest -->
  <testcase classname="org.SDET.PreacticeWIthTestNG.CustomerDependency" name="modifyCustomerTest" time="0.0">
    <ignored/>
  </testcase> <!-- modifyCustomerTest -->
  <testcase classname="org.SDET.PreacticeWIthTestNG.CustomerInvocationCount" name="deleteCustomerTest" time="0.0">
    <ignored/>
  </testcase> <!-- deleteCustomerTest -->
  <testcase classname="org.SDET.PreacticeWIthTestNG.CustomerPriority" name="deleteCustomerTest" time="0.0">
    <ignored/>
  </testcase> <!-- deleteCustomerTest -->
  <testcase classname="org.SDET.PreacticeWIthTestNG.CustomerInvocationCount" name="modifyCustomerTest" time="0.0">
    <ignored/>
  </testcase> <!-- modifyCustomerTest -->
  <testcase classname="org.SDET.PreacticeWIthTestNG.CustomerDependency" name="deleteCustomerTest" time="0.0">
    <ignored/>
  </testcase> <!-- deleteCustomerTest -->
  <testcase classname="org.SDET.PreacticeWIthTestNG.CustomerPriority" name="modifyCustomerTest" time="0.0">
    <ignored/>
  </testcase> <!-- modifyCustomerTest -->
</testsuite> <!-- Test -->
